src/popover.css

```diff
@@ -14,12 +14,22 @@
 
 /* stylelint-disable selector-class-pattern */
 @supports not selector([popover]:open) {
```

Comment:
Is it important to retain this older `:open` selector as well? I assume just temporarily, then we might remove it?

Reply:
Yeah exactly